packages/x-date-pickers-pro/src/DateRangeCalendar/DateRangeCalendar.tsx

```diff
@@ -133,12 +135,14 @@ function useDateRangeCalendarDefaultizedProps<TDate extends PickerValidDate>(
   };
 }
 
-const useUtilityClasses = (ownerState: DateRangeCalendarOwnerState<any>) => {
-  const { classes, isDragging } = ownerState;
+const useUtilityClasses = (
+  classes: Partial<DateRangeCalendarClasses> | undefined,
+  ownerState: DateRangeCalendarOwnerState,
+) => {
   const slots = {
     root: ['root'],
     monthContainer: ['monthContainer'],
-    dayCalendar: [isDragging && 'dayDragging'],
+    dayCalendar: [ownerState.isDraggingDay && 'dayDragging'],
```

Comment:
day range dragging seems to be broken on this branch. 🤷 🙈

Reply:
What do you mean by broken?
I don't see any difference with master but I might be missing something :see_no_evil:

Reply:
Sorry once again. 🙈 
Chrome on my end seems to be acting strangely and running into interesting problems. 🤯 
Reloading without the cache solved issues. 👍

Reply:
Weird that you had a bug :grimacing: